src/components/FormTextArea.vue

```diff
@@ -10,7 +10,7 @@
         @input="$emit('input', $event)"
       />
     </div>
-    <div v-else>
+    <template v-else>
```

Comment:
Since you're not wrapping multiple elements, the template isn't required. You can put the `v-else` directly on the `textarea` tag.

Reply:
yes, but <template> does not create an extra <div>, this <div> extra produces the error of the "feedback-invalid" class

![image](https://user-images.githubusercontent.com/1747025/63599921-922e9680-c590-11e9-8866-16d22035d4dd.png)

![image](https://user-images.githubusercontent.com/1747025/63599976-aecace80-c590-11e9-8821-bcd413776fd7.png)

Reply:
Right, but adding the `v-else` directly on `textarea` also doesn't create an extra `div`.

Reply:
```
<textarea
  v-else
  ...
/>
```
Gives the same result as your 1st screenshot above. You can remove the `div`, **not** add the `template` tag and, and put the `v-else` directly on the `textarea` tag.

Reply:
fixed